vendor/node-elm-compiler.js

```diff
@@ -1,9 +1,5 @@
 'use strict';
 
-/**
- * @typedef { import("../lib/types/options").Options } Options
```

Comment:
Is there a tool out there that you know of that can help finding these unused imports? Or do we already have one?

Reply:
TS can do it, but it's buggy, so I turned it back off after I turned it on.
microsoft/TypeScript#58969